src/services/completions.ts

```diff
@@ -8,7 +8,14 @@ namespace ts.Completions {
         SuggestedClassMembers = "4",
         GlobalsOrKeywords = "5",
         AutoImportSuggestions = "6",
-        JavascriptIdentifiers = "7"
+        DeprecatedLocalDeclarationPriority = "7",
+        DeprecatedLocationPriority = "8",
+        DeprecatedOptionalMember = "9",
+        DeprecatedMemberDeclaredBySpreadAssignment = "10",
+        DeprecatedSuggestedClassMembers = "11",
+        DeprecatedGlobalsOrKeywords = "12",
+        DeprecatedAutoImportSuggestions = "13",
+        JavascriptIdentifiers = "14"

```

Comment:
Just wanted to make sure javascript identifiers are below deprecated ones?
Is this ok @DanielRosenwasser ?